src/main/java/com/dev/cinema/mapper/MovieSessionMapper.java

```diff
@@ -0,0 +1,42 @@
+package com.dev.cinema.mapper;
+
+import com.dev.cinema.dto.moviesession.MovieSessionRequestDto;
+import com.dev.cinema.dto.moviesession.MovieSessionResponseDto;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.MovieService;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapper {
+    private final MovieService movieService;
+    private final CinemaHallService hallService;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd HH:mm");
```

Comment:
should it be a constant field?

Reply:
I am not sure. I did it for the purpose to be sure that all saved dates and times will be in the same format. It is possible to make a mix of different types of format if not use it. (2020.07.07 or 2020-07-07 etc.) Do I have to remove it?

Reply:
just make it a constant field :)